pilot/pkg/config/kube/gateway/testdata/route-precedence.status.yaml.golden

```diff
@@ -74,38 +74,6 @@ metadata:
 spec: null
 status:
   parents:
-  - conditions:
```

Comment:
this change looks unexpected, even considering the issue I mentioned on the other test. It looks like we have a route with 3 parents, and 2 are not found, but we end up only reporting a status for the valid one.

probably this test should be returned to "3 valid parents" but we should add an additional test case with a mix of invalid and valid parents?

Reply:
Yea I think this is not intentional. Adding the missing services to the conversion unit tests did not revert this diff so something else is going on. I will update this test case to use 3 valid parents and another test with the mix of invalid and valid parents.